index.in.html

```diff
@@ -139,9 +152,14 @@ <h3>Building universal binaries</h3>
 <a name=pkg></a>
 <h2>Experimental binary packages</h2>
 
-This site is also a repository for experimental binaries, i.e. you could set it as your default repository for development verion of R such as <tt>options(repos=c(CRAN="http://r.research.att.com/"))</tt>
+This site is also a repository for experimental binaries, i.e. you could set it as your default repository for development verion of R such as 
+
+<div style='padding-top:1rem; padding-left:1rem'>
+  <tt>options(repos=c(CRAN="http://r.research.att.com/"))</tt>
```

Comment:
Should this link be updated to:

```suggestion
  <tt>options(repos=c(CRAN="https://mac.r-project.org/"))</tt>
```

Reply:
```r
# Note: -17 removes "Repository Field"
new_site = 
   available.packages(contrib.url("https://mac.r-project.org"))[,-17] 

old_site = 
   available.packages(contrib.url("http://r.research.att.com"))[,-17]

all.equal(new_site, old_site)
# TRUE
```

Though, the redirect seems to be going to:

```
https://cloud.r-project.org/src/contrib/PACKAGES
```

Maybe this feature was removed?

Reply:
Looks like that might be the case. I'll make the switch regardless (since relying on a AT&T redirect prbly shld not be a thing) and make ^^ an issue to track after.